drf_spectacular/views.py

```diff
@@ -64,8 +79,11 @@ class SpectacularSwaggerView(APIView):
 
     @extend_schema(exclude=True)
     def get(self, request, *args, **kwargs):
+        schema_url = self.url or reverse(self.url_name, request=request)
+        if request.GET.get('lang'):
+            schema_url += f'?lang={request.GET.get("lang")}'
```

Comment:
`?` may already be in schema_url; if so, need to use `&` instead

Reply:
done